src/components/Filters.js

```diff
@@ -53,7 +53,7 @@ const Filters = () => {
                     name="category"
                     type="button"
                     className={`${
-                      categories === category.toLowerCase() ? 'active' : null
```

Comment:
Currently this is checking if the **array** `categories` ever matches the current `category` being iterated on; but that is always false so the `"active"` class is never applied.